docs/api-reference/tools/index.rst

```diff
@@ -1,11 +1,12 @@
 .. _tools:
 
-=====================================
-Command line tools (`~ctapipe.tools`)
-=====================================
+*************************************
+Command-Line Tools (`~ctapipe.tools`)
```

Comment:
In `docs/api-reference/reco/index.rst` you capitalise hyphenated words ("Moment-based") as "Capital-lowercase"  but here you write "Command-Line", that is using a "Capital-Capital" format. I think you chose what format to use and use that consistently.

Reply:
Oh, you're right. I decided to stick with "Capital-lowercase" when going through the files, but I guess I missed that one.